src/components/BranchMenu.tsx

```diff
@@ -314,7 +314,41 @@ export class BranchMenu extends React.Component<
      * @param err - error
      */
     function onError(err: any): void {
-      showErrorMessage('Error switching branch', err.message);
+      if (err.message.includes('following files would be overwritten')) {
+        showDialog({
+          title: 'Unable to switch branch',
+          body: (
+            <React.Fragment>
+              <p>
+                Your changes to the follow files would be overwritten by
```

Comment:
oops, just noticed this grammar error. In that this was just merged whats the best way to fix this?
```suggestion
                Your changes to the following files would be overwritten by
```